### PR TITLE
ARS-421: Describe the goods

### PR DESCRIPTION
**Pull request by C-Thomp**
スクリーンショット 0005-01-11 16 40 03

### CODE REVIEW

File: app/viewmodels/govuk/TextareaFluency.scala [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import play.api.data.Field
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.viewmodels.hint.Hint
+import uk.gov.hmrc.govukfrontend.views.viewmodels.label.Label
+import uk.gov.hmrc.govukfrontend.views.viewmodels.textarea.Textarea
+import viewmodels.ErrorMessageAwareness
+
+object textarea extends TextareaFluency
+
+trait TextareaFluency {
```
Comment (RawToast) [inline app/viewmodels/govuk/TextareaFluency.scala:28]: 🎉 
I'd append this to `viewmodels.package.scala` similar to other extension methods 👍

**Review by RawToast — COMMENTED** [review]
We fixed the bobby warning in #22 which might be the cause of your build failures

File: app/views/DescribeTheGoodsView.scala.html [hunk 2]
```diff
@@ -14,36 +14,60 @@
...
+        @govukTextarea(
+            TextareaViewModel(
                 field = form("value"),
-                label = LabelViewModel(messages("describeTheGoods.heading")).asPageHeading()
-            )
-            .withWidth(Full)
+                label = Label(None)
+                )
         )
+<!--        <textarea class="govuk-textarea" id="value" name="description-box" rows="15" aria-describedby="description-of-goods"></textarea>-->
```
Comment (RawToast) [inline app/views/DescribeTheGoodsView.scala.html:66]: Do we still need this line?

File: app/views/DescribeTheGoodsView.scala.html [hunk 3]
```diff
@@ -14,36 +14,60 @@
...
 
         @govukButton(
             ButtonViewModel(messages("site.continue"))
         )
     }
+    @cancelLink()
```
Comment (RawToast) [inline app/views/DescribeTheGoodsView.scala.html:72]: 🥳

**Review by RawToast — APPROVED** [review]

File: app/views/DescribeTheGoodsView.scala.html (hunk 2 above)
Comment (C-Thomp) [reply app/views/DescribeTheGoodsView.scala.html:66]: Absolutely not! 🔧

**Review by RawToast — APPROVED** [review]

**Pull request merged**